wt-status.c

```diff
@@ -1751,5 +1844,595 @@ void wt_porcelain_print(struct wt_status *s)
 	s->relative_paths = 0;
 	s->prefix = NULL;
 	s->no_gettext = 1;
-	wt_shortstatus_print(s);
+
+	if (s->verbose > 1)
+		wt_porcelain_very_verbose_print(s);
+	else
+		wt_shortstatus_print(s);
+}
+
+/*****************************************************************
+ * Very Verbose Porcelain Output Routines.
+ *****************************************************************/
+
+static int wt_vvp_count_unmerged(struct wt_status *s)
+{
+	int sum = 0;
+	int i;
+
+	for (i = 0; i < s->change.nr; i++) {
+		struct wt_status_change_data *d;
+		d = s->change.items[i].util;
+		if (d->stagemask)
+			sum++;
+	}
+	return sum;
+}
+
+/* Lookup rebase work completed and still necessary.
+ * The essence of show_rebase_information().
+ */
+static void wt_vvp_count_rebase(
+	struct wt_status *s,
+	int *nr_done,
+	int *nr_todo)
+{
+	struct string_list have_done = STRING_LIST_INIT_DUP;
+	struct string_list yet_to_do = STRING_LIST_INIT_DUP;
+
+	/* This is a bit expensive way to count the lines
+	 * in these 2 files.  We might simplify this after
+	 * we decide we don't want to print the line data.
+	 */
+	read_rebase_todolist("rebase-merge/done", &have_done);
```

Comment:
Please note that this file does not exist while editing the initial script for `rebase -i`, and we `die()` here if it does not exist. Maybe be safe and guard it behind a `file_exists()`?